contracts/contracts/strategies/NativeStaking/ValidatorAccountant.sol

```diff
@@ -170,66 +158,65 @@ abstract contract ValidatorAccountant is ValidatorRegistrator {
                 ethRemaining
             );
         }
-        // Oh no... Fuse is blown. The governor (Multisig not OGV Governor) needs to set the
-        // record straight by manually set the accounting values.
+        // Oh no... Fuse is blown. The Strategist needs to adjust the accounting values.
         else {
-            // will emit a paused event
-            _pause();
+            // pause if not already
+            if (!paused()) {
+                _pause();
+            }
+            // fail the accounting
             accountingValid = false;
         }
     }
 
-    /// @dev allow the accounting governor to fix the accounting of this strategy and unpause
-    /// @param _activeDepositedValidators the override value of activeDepositedValidators
-    /// @param _ethToWeth the amount of ETH to be converted to WETH
-    /// @param _wethToBeSentToVault the amount of WETH to be sent to the Vault
-    /// @param _consensusRewards the override value for consensusRewards
-    /// @param _ethThresholdCheck maximum allowed ETH balance on the contract for the function to run
-    /// @param _wethThresholdCheck maximum allowed WETH balance on the contract for the function to run
-    ///        the above 2 checks are done so transaction doesn't get front run and cause
-    ///        unexpected behaviour
+    /// @notice Allow the Strategist to fix the accounting of this strategy and unpause.
+    /// @param _validatorsDelta adjust the active validators by plus one, minus one or unchanged with zero
+    /// @param _wethToVaultAmount the amount of WETH to be sent to the Vault
+    /// @param _consensusRewardsDelta adjust the accounted for consensus rewards up or down
     function manuallyFixAccounting(
-        uint256 _activeDepositedValidators,
-        uint256 _ethToWeth,
-        uint256 _wethToBeSentToVault,
-        uint256 _consensusRewards,
-        uint256 _ethThresholdCheck,
-        uint256 _wethThresholdCheck
-    ) external onlyAccountingGovernor whenPaused {
-        uint256 ethBalance = address(this).balance;
-        uint256 wethBalance = IWETH9(WETH_TOKEN_ADDRESS).balanceOf(
-            address(this)
+        int256 _validatorsDelta,
+        int256 _consensusRewardsDelta,
+        uint256 _wethToVaultAmount
+    ) external onlyStrategist whenPaused {
+        require(
+            _validatorsDelta >= -1 &&
+                _validatorsDelta <= 1 &&
```

Comment:
the -1 -> 1 accepted validator delta and -32 -> 32 accepted consensus delta could be constants so that it is easier to configure these values if/when we want. 

Also if these ranges are not able to fix the accounting we need to re-deploy the contract via OUSD governance (slow process) to introduce larger deltas. I think we should be comfortable changing up to 3 or even 5 validators (and 3*32 or 5*32 ETH in consensus rewards) without being worried that such transaction could rug anyone or produce so much rewards for the attacker (controlling the strategist) to somehow damage the protocol.

Reply:
Thanks, I'll update to 2 and 332